src/app/AppLayout/AppLayout.tsx

```diff
@@ -421,14 +417,15 @@ export const AppLayout: React.FC<AppLayoutProps> = ({ children }) => {
                     toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
                       <MenuToggle
                         ref={toggleRef}
+                        variant="plainText"
                         icon={<QuestionCircleIcon />}
                         className="application-launcher"
                         onClick={() => handleHelpToggle()}
-                      >
-                        <EllipsisVIcon />
-                      </MenuToggle>
+                      ></MenuToggle>
```

Comment:
How about this?

```diff
diff --git a/src/app/AppLayout/AppLayout.tsx b/src/app/AppLayout/AppLayout.tsx
index e61d3e4..b18d802 100644
--- a/src/app/AppLayout/AppLayout.tsx
+++ b/src/app/AppLayout/AppLayout.tsx
@@ -407,7 +407,9 @@ export const AppLayout: React.FC<AppLayoutProps> = ({ children }) => {
                     data-quickstart-id="settings-link"
                     component={(props) => <Link {...props} to="/settings" />}
                   >
-                    <CogIcon />
+                    <Icon>
+                      <CogIcon />
+                    </Icon>
                   </Button>
                 </ToolbarItem>
                 <ToolbarItem>
@@ -417,11 +419,14 @@ export const AppLayout: React.FC<AppLayoutProps> = ({ children }) => {
                     toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
                       <MenuToggle
                         ref={toggleRef}
-                        variant="plainText"
-                        icon={<QuestionCircleIcon />}
+                        variant="plain"
                         className="application-launcher"
                         onClick={() => handleHelpToggle()}
-                      ></MenuToggle>
+                      >
+                        <Icon>
+                          <QuestionCircleIcon />
+                        </Icon>
+                      </MenuToggle>
                     )}
                     isOpen={showHelpDropdown}
                     onOpenChange={(v) => setShowHelpDropdown(v)}
```

Reply:
This should render the question icon only with the "down caret", similar to the old `2.x`.

![image](https://github.com/user-attachments/assets/4be96687-0cbf-4064-a0c9-c35ee42647eb)